misc/alert_rules.json

```diff
@@ -798,5 +798,32 @@
     "rule": "applications.app_type = \"cape\" && application_metrics.metric = \"error\" && application_metrics.value > 0",
     "name": "CAPE Errors > 0",
     "severity": "critical"
+  },
+  {
+    "builder": {"condition":"AND","rules":[
+                                           {"id":"applications.app_type","field":"applications.app_type","type":"string","input":"text","operator":"equal","value":"smart"},
+                                           {"id":"application_metrics.metric","field":"application_metrics.metric","type":"string","input":"text","operator":"regex","value":"disk.*fail.*"},
+                                           {"id":"application_metrics.value","field":"application_metrics.value","type":"string","input":"number","operator":"greater_or_equal","value":"1"}
+                                         ],"valid":false},
+    "name": "SMART: one or more disk is has failures",
+    "severity": "critical"
+  },
+  {
+    "builder": {"condition":"AND","rules":[
+                                           {"id":"applications.app_type","field":"applications.app_type","type":"string","input":"text","operator":"equal","value":"smart"},
+                                           {"id":"application_metrics.metric","field":"application_metrics.metric","type":"string","input":"text","operator":"regex","value":"disk.*readfailure"},
```

Comment:
Won't this one get picked up by rule one based on the regex?

Reply:
The first will catch either. The second and third are if some one would like alerts that specify which one instead one one that is more generic.

Using the first with a value of 4 here at work here... HP drives seem to occasional produce sporadic failures on long self tests and seems to only be notable if it is repeatedly failing. :/